doc/tutorials/run_kata_containers.rst

```diff
@@ -19,14 +19,15 @@ Prerequisites
 #. Refer to the :ref:`ACRN supported hardware <hardware>`.
 #. For a default prebuilt ACRN binary in the E2E package, you must have 4
    CPU cores or enable "CPU Hyper-Threading" in order to have 4 CPU threads for 2 CPU cores.
-#. Follow :ref:`these instructions <Ubuntu Service OS>` to set up the ACRN Service VM
+#. Follow the :ref:`rt_industry_ubuntu_setup` to set up the ACRN Service VM
    based on Ubuntu.
 #. This tutorial is validated on the following configurations:
 
-   - ACRN v1.6.1 (tag: acrn-2020w18.4-140000p)
-   - Ubuntu 18.04.4
+   - ACRN v2.0 (branch: ``release_2.0``)
+   - Ubuntu 20.04
+
 #. Kata Containers are only supported for ACRN hypervisors configured for
-   the SDC scenario.
+   the Industry scenario.
```

Comment:
@gvancuts , per [[1]](https://github.com/projectacrn/acrn-hypervisor/blob/release_2.0/hypervisor/scenarios/sdc/vm_configurations.c#L51-L64) and [[2]](https://github.com/projectacrn/acrn-hypervisor/blob/release_2.0/hypervisor/scenarios/industry/vm_configurations.c#L123), it looks like both SDC and Industry scenarios are supported.
The rest of modifications look good, and just learned `kata-deploy` from you! Thank you (y)

Reply:
Very good point, I had missed that completely!

(note to self: I should also update the https://projectacrn.github.io/latest/introduction/index.html#best-known-configurations table)

Reply:
> (note to self: I should also update the https://projectacrn.github.io/latest/introduction/index.html#best-known-configurations table)

See #4950